app/models/client_opportunity_match.rb

```diff
@@ -884,6 +884,10 @@ def similar_matches
       end
   end
 
+  def associated_file_tags
+    sub_program.file_tags.pluck(:tag_id) + requirements_with_inherited.map(&:associated_file_tags).flatten.compact
```

Comment:
I wonder if it makes sense to expose which thing is requiring a particular tag?  As we do when we expose the rules and expose the source of the requirement. Something like:

State-issued id _(program name)_
DD-214 _(Vet Funding Source)_

Reply:
Like so?
<img width="250" alt="image" src="https://user-images.githubusercontent.com/33072305/210596736-218c8559-23ab-43a8-a5ef-4269f507ded6.png">

Reply:
Exactly.  Thanks!